### PR TITLE
JAMES-2505 remove manual TmpFs management

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

File: backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java [hunk 1]
```diff
@@ -84,16 +66,16 @@ public DockerCassandraRule() {
                         .run("sed -i -e \"s/commitlog_sync_period_in_ms: 10000/commitlog_sync_period_in_ms: 9999999/\" " + CASSANDRA_YAML)
                         //auto_bootstrap should be useless when no existing data
                         .run("echo auto_bootstrap: false >> " + CASSANDRA_YAML)
-                        .run("echo \"-Xms1500M\" >> " + JVM_OPTIONS)
-                        .run("echo \"-Xmx1500M\" >> " + JVM_OPTIONS)
+                        .run("echo \"-Xms500M\" >> " + JVM_OPTIONS)
+                        .run("echo \"-Xmx500M\" >> " + JVM_OPTIONS)
```
Comment (chibenwa) [inline backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java]: Downgrading memory don't seem related with the title of the commit. Could you revert this?

File: backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java [hunk 2]
```diff
@@ -84,16 +66,16 @@ public DockerCassandraRule() {
...
                         // disable assertions (modest performance benefit)
                         .run("sed -i -e 's/JVM_OPTS=\"$JVM_OPTS -ea\"/JVM_OPTS=\"$JVM_OPTS -da\"/' " + CASSANDRA_ENV)
                         // use caches for keys & rows
-                        .run("sed -i -e \"s/key_cache_size_in_mb:/key_cache_size_in_mb: 256/\" " + CASSANDRA_YAML)
-                        .run("sed -i -e \"s/row_cache_size_in_mb: 0/row_cache_size_in_mb: 512/\" " + CASSANDRA_YAML)
+                        .run("sed -i -e \"s/key_cache_size_in_mb:/key_cache_size_in_mb: 50/\" " + CASSANDRA_YAML)
+                        .run("sed -i -e \"s/row_cache_size_in_mb: 0/row_cache_size_in_mb: 50/\" " + CASSANDRA_YAML)
```
Comment (chibenwa) [inline backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java]: Idem for downgrading cache sizes.

File: backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java [hunk 3]
```diff
@@ -84,16 +66,16 @@ public DockerCassandraRule() {
...
                         .build()))
-            .withCreateContainerCmdModifier(cmd -> cmd.getHostConfig().withBinds(new Binds(new Bind(tmpFsName, new Volume("/var/lib/cassandra")))))
-            .withCreateContainerCmdModifier(cmd -> cmd.withMemory(2000 * 1024 * 1024L))
+            .withCreateContainerCmdModifier(cmd -> cmd.withMemory(800 * 1024 * 1024L))
```
Comment (chibenwa) [inline backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java]: Idem for limiting memory allocated to the container.

**Comment by chibenwa** [comment]
test this please (build output lost)

**Comment by chibenwa** [comment]
```
[2ef96aec43d9ebe1fb52b22fec1a6c9a51848b6c] Failed tests: 
[2ef96aec43d9ebe1fb52b22fec1a6c9a51848b6c]   MailRepositoryContract.removeShouldHaveNoEffectForUnknownMails:333 
[2ef96aec43d9ebe1fb52b22fec1a6c9a51848b6c] Expecting empty but was:<[MailKey{value=mail1}]>
[2ef96aec43d9ebe1fb52b22fec1a6c9a51848b6c] Tests in error: 
[2ef96aec43d9ebe1fb52b22fec1a6c9a51848b6c]   MailRepositoryContract.retrieveShouldReturnAllMailProperties » Completion com....
[2ef96aec43d9ebe1fb52b22fec1a6c9a51848b6c]   MailRepositoryContract.storeBigMailShouldNotFail » Completion com.datastax.dri...
```

File: backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java (hunk 1 above)
Comment (mbaechler) [reply backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java]: oh yes, it was to make it run on my computer

**Comment by mbaechler** [comment]
merged
